notebooks/117-optimize-preprocessing/117-optimize-preprocessing.ipynb

```diff
@@ -0,0 +1,498 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.                             .mean([104.0, 117.0, 123.0])
</pre><p>can be these commented steps described as part of markdown?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/811/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
moved